<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>741</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>直线电机测试</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>控制器设置</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_init">
         <property name="text">
          <string>初始化控制器</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_axis_enable">
         <property name="text">
          <string>使能开启</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_axis_disable">
         <property name="text">
          <string>使能关闭</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_close">
         <property name="text">
          <string>关闭控制器</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_2" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>速度运转</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_5" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>速度：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="text_speed">
            <property name="text">
             <string>2</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>mm/s</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_6" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="btn_sp_z">
            <property name="text">
             <string>正向</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_sp_t">
            <property name="text">
             <string>停止</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_sp_home">
            <property name="text">
             <string>回零</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_sp_f">
            <property name="text">
             <string>反向</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_3" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>位置运转</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_7" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>位置：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="text_pos">
            <property name="text">
             <string>5</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>mm</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_8" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QPushButton" name="btn_pos_z">
            <property name="text">
             <string>正向</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_pos_t">
            <property name="text">
             <string>停止</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_pos_home">
            <property name="text">
             <string>回零</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_pos_f">
            <property name="text">
             <string>反向</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_4" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>循环测试</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_9" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>步长：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="text_pos_c">
            <property name="text">
             <string>5</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>mm</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>速度：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="text_speed_c">
            <property name="text">
             <string>2</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_12">
            <property name="text">
             <string>mm/s</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_10" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QLabel" name="label_13">
            <property name="text">
             <string>循环点数：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="text_point_c">
            <property name="text">
             <string>1</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="label_14">
            <property name="text">
             <string>循环次数：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="text_times_c">
            <property name="text">
             <string>1</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_12" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <widget class="QLabel" name="label_15">
            <property name="text">
             <string>剩余循环点数：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_points">
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_16">
            <property name="text">
             <string>剩余循环次数：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_times">
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_11" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QPushButton" name="btn_ct_z">
            <property name="text">
             <string>正向</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_ct_t">
            <property name="text">
             <string>停止</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_ct_home">
            <property name="text">
             <string>回零</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_ct_f">
            <property name="text">
             <string>反向</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_13" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <item>
        <widget class="QLabel" name="label_17">
         <property name="text">
          <string>轴状态：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="axis_status">
         <property name="text">
          <string>未启动</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_19">
         <property name="text">
          <string>编码器位置：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="coder_pos">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_18">
         <property name="text">
          <string>mm</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
